src/Language/Haskell/LSP/Control.hs

```diff
@@ -8,40 +8,54 @@
 module Language.Haskell.LSP.Control
   (
     run
-  , sendNotificationMessage
-  , sendRequestMessage
-  , sendResponseMessage
+  -- , sendNotificationMessage
```

Comment:
In hie I store the `sendFunc` returned as the first parameter of a `Handler`, and use that.
Perhaps we should extend the `Control.run` callback function to be

```haskell
(J.ClientCapabilities -> (BSL.ByteString -> IO ()) -> IO (Maybe J.ResponseError))
```

so that the second parameter is the send function?